rag/brentford_craw.py

```
@@ -0,0 +1,51 @@
+import pandas as pd
+import os
+
+headers = {"User-Agent": "Mozilla/5.0"}
+BASE_URL = "https://www.transfermarkt.com/brentford-fc/rekordspieler/verein/1148/wettbewerb_id/alle/position/alle/aktive/alle/detailposition/alle/page/{}"
+
+def scrape_brentford_records(max_page=26):
+    players_data = []
+
+    for page in range(1, max_page+1):
+        url = BASE_URL.format(page)
+        res = requests.get(url, headers=headers)
```

Ruff (0.13.2)

Probable use of `requests` call without timeout

(S113)